vector/src/main/java/im/vector/app/core/pushers/PushersManager.kt

```diff
@@ -64,14 +65,16 @@ class PushersManager @Inject constructor(
             gateway: String
     ) = HttpPusher(
```

Comment:
Is it possible to set all fields with name (like you did for `appId` for example) to avoid any confusion?